src/components/common/header/AuthenticatedHeader.js

```diff
@@ -1,23 +1,30 @@
-import React, { Fragment, useEffect } from 'react';
-import { Link } from 'react-router-dom';
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
I don't see the need for this anymore

Reply:
done...Though I had to put a dummy path in Link, instead of leaving it empty, unless eslint would still flag it.
so I have something like `<Link to='#*' onClick={logout(....)}>.....`